sell-loot.lic

```diff
@@ -50,23 +68,13 @@ class SellLoot
     if @bankbot_enabled
       give_money_to_bankbot(@local_currency, @bankbot_deposit_threshold)
     else
-      deposit_coins(keep_coppers_bank)
+      deposit_coins(keep_money_by_currency)
     end
   end
 
   def parse(setting)
-    arg_definitions = [
-      [
-        { name: 'amount', regex: /\d+/i, variable: true, description: 'Number of coins to keep' },
```

Comment:
❓ Why did you delete these args? I use them sometimes...

Reply:
If you call `;sell-loot 1 plat`, then (in the old code) the args are checked _before_ your settings. I use it when I want to end up with a certain amount on hand which is different than my setting.